python/cudf/cudf/tests/test_pandas_comparison.py

```diff
@@ -0,0 +1,366 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+import inspect
+import textwrap
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import cudf
+from cudf.testing._utils import assert_eq
+
+
+def _is_cudf(lib):
+    # TODO: Is it safe to use `if lib is cudf`? I think there are some cases
```

Comment:
Follow up on this TODO. If you can't find conclusive proof one way or the other, I am happy with `lib.__name__ == "cudf"`.